R/get_flagged_records.R

```diff
@@ -0,0 +1,104 @@
+#' Create an Existence Flag
+#'
+#' @description Create a flag variable for the input dataset which indicates if
+#' there exists at least one observation in another dataset fulfilling a certain
+#' condition.
+#'
+#' **Note:** This is a helper function for `derive_vars_merged_exist_flag` which
+#' inputs this result into `derive_vars_merged()`.
+#'
+#' @param dataset The additional dataset you wish to pull variables from
+#'
+#' @param new_var New variable
+#'
+#'   The specified variable is added to the input dataset.
```

Comment:
I see the new variable is always coded to 1 or 0. @bundfussr should we specify this in the argument on the return value?

Reply:
I don't think we have a convention where this should be specified. For me either way is fine.